pytools/__init__.py

```diff
@@ -1597,6 +1597,32 @@ def github_markdown(self):
 
         return "\n".join(lines)
 
+    def csv(self, dialect='excel', **fmtparams):
```

Comment:
Document `fmt_params` (or maybe `csv_kwargs`), e.g. by linking to `:func:\`csv.writer\``.

Reply:
Added.